src/commands/generate/inspec_profile.ts

```diff
@@ -469,8 +452,8 @@ For more information on developing overlays, reference the [MITRE SAF Training](
 >Inputs are variables that are referenced by control(s) in the profile that implement them.
  They are declared (defined) and given a default value in the \`inspec.yml\` file. 
 
-#### Update Profile Inputs
-Inputs can be overridden by providing an input file or a CLI flag at execution time.
+#### Using Inputs from the CLI or Local File
+Your tailored inputs can be providing an input file or a CLI flag at execution time.
```

Comment:
How about "Your tailored inputs may be used by". The overridden term reminds too much of "waiver" or "recast", a connotation of using inputs to avoid compliance. please?